assets/src/web-stories-block/block/components/blockTypeSwitcher.js

```diff
@@ -23,57 +23,34 @@ import PropTypes from 'prop-types';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import {
-  DropdownMenu,
-  ToolbarGroup,
-  Toolbar,
-  ToolbarItem,
-} from '@wordpress/components';
+import { DropdownMenu, ToolbarGroup, ToolbarItem } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import { BLOCK_TYPES } from '../constants';
 
 function BlockTypeSwitcher({ selectedBlockType, setAttributes }) {
-  // Note: ToolbarGroup and ToolbarButton are only available in Gutenberg 7.0 or later,
-  // so they do not exist in WP 5.3.
-  const ToolbarComponent = ToolbarGroup ? ToolbarGroup : Toolbar;
```

Comment:
Do `wp.components.ToolbarGroup` and `wp.components.ToolbarButton` exist in 5.5 already?

The e2e test failures seem to indicate otherwise.

Reply:
Looks like it was added 5.6 - https://make.wordpress.org/core/2020/11/18/changes-to-toolbar-components-in-wordpress-5-6/ 🤦